library/waku_thread/inter_thread_communication/requests/node_lifecycle_request.nim

```diff
@@ -1,11 +1,5 @@
-
-import
-  std/options
-import
-  chronos,
-  chronicles,
-  stew/results,
-  stew/shims/net
+import std/options
+import chronos, chronicles, stew/results, stew/shims/net
```

Comment:
It's been commented on elsewhere but this should be vertical IMO

Reply:
you can make it vertical in several ways:
* use a single import statement that gets "too long" and "too complex"
* document the imports:

```
import 
  chronos # bla bla
  abc # bla bla
```

when imports are "simple", we use a simple formatting as well (many in the community, confirmed by looking at how imports are formatted in the wild), prefer not to have them spread out too much, when it's just a few